### DISCUSSION TITLE
TC-RMW configuration

### DISCUSSION THREAD

**Discussion by KathrynNewman** [dtcenter/METplus#1451] [Configuration] [answered]
Hi! I'm finding that tangential and radial winds are not being calculated in my TC-RMW output. In 25.1 of the MET Users' Guide it states that if wind fields are specified in the configuration file, the radial and tangential wind components will be computed. I have UGRD/VGRD specified in my config file - is there something else that needs to be done in order to get this output?

Thanks,
Kathryn

**Comment by CPKalb** [comment, marked as answer]
Hi Kathryn:

I'm following up, as it looks like this is a bug and we have an issue out for a fix (https://github.com/dtcenter/MET/issues/2072).   Is it okay to close this discussion?

Thanks,
Christina

**Reply by KathrynNewman** [reply]
Thanks for following up! Yes, please go ahead and close as an issue has been created.
